airflow/www/fab_security/manager.py

```diff
@@ -1397,6 +1424,24 @@ def get_user_menu_access(self, menu_names: list[str] | None = None) -> set[str]:
         else:
             return self._get_user_permission_resources(None, "menu_access", resource_names=menu_names)
 
+    def add_limit_view(self, baseview):
```

Comment:
I don't think we expose that view as we are not exposing other views, but I added it anyway.